src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -28,7 +28,7 @@ public class FileExpressionEvaluationTests : PowerFxTest
         // See the README.md in the ExpressionTestCases directory for more details.
 
         [InterpreterTheory]
-        [TxtFileData("ExpressionTestCases", "InterpreterExpressionTestCases", nameof(InterpreterRunner), "TableSyntaxDoesntWrapRecords,ConsistentOneColumnTableResult,NumberIsFloat")]
+        [TxtFileData("ExpressionTestCases", "InterpreterExpressionTestCases", nameof(InterpreterRunner), "TableSyntaxDoesntWrapRecords,ConsistentOneColumnTableResult,NumberIsFloat,DecimalSupport")]
```

Comment:
Shouldn't this be run without DecimalSupport to test the compiler as Canvas would use it?  Or is DecimalSupport an interpreter only setting?  Either way, it would be good to comment as this looks a little funny for now.

Reply:
It's an interpreter support flag. Added a comment to clarify it.